code/mob/living/critter/ai/ocean.dm

```diff
@@ -175,8 +173,7 @@
 
 /datum/aiTask/timed/targeted/flee_and_shoot/frustration_check()
 	.= 0
-	var/dist = get_dist(holder.owner, holder.target)
-	if (dist >= target_range)
+	if (!DIST_CHECK(holder.owner, holder.target, target_range))
```

Comment:
Technically not equivallent as your version is basically `>` and original code was `>=` but it *probably* doesn't matter?

Reply:
@mybluecorners does this matter?